packages/block-editor/src/components/button-block-appender/styles.native.scss

```diff
@@ -0,0 +1,17 @@
+.appender {
+	align-items: center;
+	justify-content: center;
+	background-color: $gray-light;
+	padding: 12px;
+	background-color: $white;
```

Comment:
@lukewalczak - I noticed that here `background-color` is defined two times.
It's a small detail but it was giving a CI error on this PR: https://github.com/WordPress/gutenberg/pull/17109

It's interesting that after re-running CI it now succeeded, but the issue is real. So it would be nice if we could fixe it.
I also wonder why it was catch once on another branch but it wasn't catch on this PR, and locally it also succeed 🤔 

Would you mind taking a look? Thanks!

cc @pinarol

Reply:
Is there an option to remove redundant bg color `background-color: $gray-light;` within mentioned PR?

Reply:
It is unclear when that PR will be merged, so it'd be better if we open a new PR to remove the redundant line. We can get the new one merged quickly.

Reply:
Ok, will create a PR for that.

Reply:
https://github.com/WordPress/gutenberg/pull/17325